analytics/src/views/Dashboard/components/Map/Map.js

```diff
@@ -44,38 +44,103 @@ const Map = props => {
 
   const classes = useStyles();
 
+  const [contacts,setContacts ] = useState([]);
+
+  useEffect(() => {
+    fetch('https://analytcs-bknd-service-dot-airqo-250220.uc.r.appspot.com/api/v1/dashboard/monitoringsites?organisation_name=KCCA')
```

Comment:
Could we add all these awesome constants to one location, ie: `analytics/src/config/constants.js`?